doc/man1/openssl-genpkey.pod.in

```diff
@@ -19,7 +19,7 @@ B<openssl> B<genpkey>
 [B<-verbose>]
 [B<-quiet>]
 [B<-pass> I<arg>]
-[B<-I<cipher>>]
+[B<-cipher>]
```

Comment:
I think the original version was correct. The `I` here is deliberate to indicate that you are not supposed to pass the literal string`-cipher`, but instead replace the word `cipher` with an actual cipher name